.github/workflows/release.yml

```diff
@@ -0,0 +1,54 @@
+name: Release pipeline
+
+on:
+  release:
+    types:
+      - released
+    tags:
+      - 'v*'
+
+env:
+  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  INTEGRATION: "mysql"
+  ORIGINAL_REPO_NAME: 'newrelic/nri-mysql'
```

Comment:
Can we get these two from the github api having it dynamic? I fear seeing mysql copied pasted everywhere 😅

Reply:
maybe? the repo name might be easy, the Integration will need some work. 
But yes, these 2 should definitely be "calculated" and not copy-pasted

Reply:
I'll see if I can do it